lgsm/functions/info_messages.sh

```diff
@@ -636,6 +636,20 @@ fn_info_message_ark(){
 	} | column -s $'\t' -t
 }
 
+fn_info_message_avorion() {
```

Comment:
ports need clarifying

Reply:
well, the rcon port for example is only enabled if you set a password in the config file.
As default it is in serverfiles/galaxy/LinuxGSM/server.ini

example with rcon enabled:
```
tcp        0      0 1.2.3.4:27015   0.0.0.0:*               LISTEN      20360/./bin/Avorion 
udp        0      0 1.2.3.4:27008   0.0.0.0:*                           20360/./bin/Avorion 
udp        0      0 1.2.3.4:27020   0.0.0.0:*                           20360/./bin/Avorion 
```

from the log of the server (located in serverfiles/galaxy/LinuxGSM/):
```
Game Port: 27000
Steam Port: 27021
Steam Query Port: 27020
Query Port: 27008
```

if disabled, it is also noted there in the log:
`Warning: No RCON password set. RCON disabled.`

Reply:
anyway the 3rd gameport does not seem to be right anymore, i will exchnage this one with the queryport.